apps/dashboard/src/api/workflows.ts

```diff
@@ -1,5 +1,5 @@
-import type { WorkflowResponseDto } from '@novu/shared';
 import { getV2 } from './api.client';
+import { WorkflowResponseDto } from '@novu/shared';
```

Comment:
We specifically added the import type from shared to fix some issues with the new dashboard project's linting, building, and HMR tasks.

Why did we remove it now? @LetItRock can you please chip-in?

Reply:
reverting